public/js/thirdpartyservices.js

```diff
@@ -92,5 +92,15 @@ var thirdPartyServicesJsonArray = [
       privacy: 'https://clicky.com/terms#privacy',
       terms: ''
     }
+  },
+  {
+    name: 'Flurry Analytics',
+    model: 'flurry',
+    flurry: false,
+    logo: 'images/flurry.png',
+    link: {
+      privacy: 'https://developer.yahoo.com/flurry/legal-privacy/terms-service/',
+      terms: ''
```

Comment:
From what I can check online, flurry's terms and conditions link is this

https://developer.yahoo.com/flurry/legal-privacy/terms-service/flurry-analytics-terms-service.html

Reply:
@Tyrbok you can also use the link I added above and set it for `terms` key in the json, right below `privacy`.

Reply:
Sure! 

(I didn't see where in the code is used)